docs/Customization/Firrtl-Transforms.rst

```diff
@@ -8,21 +8,27 @@ modify Chisel-elaborated RTL.
 As mentioned in Section :ref:`Tools/FIRRTL:firrtl`, transforms are modifications that happen on the FIRRTL IR that can modify a circuit.
 Transforms are a powerful tool to take in the FIRRTL IR that is emitted from Chisel and run analysis or convert the circuit into a new form.
 
+The Scala FIRRTL Compiler and the MLIR FIRRTL Compiler
+------------------------------------------------------
+In Chipyard, two FIRRTL compilers work together to compile Chisel into Verilog. The Scala FIRRTL compiler(SFC) and the MLIR FIRRTL compiler(MFC).
+They are basically doing the same thing, except that MFC is written in C++ which makes compilation much faster. In the default setting, the SFC will compile Chisel into CHIRRTL and MFC will
```

Comment:
Can you add some text describing why we still need the SFC passes to run? Which passes are SFC-only?

Reply:
Come to think of it, I'm not sure why we still need SFC. Is it because MFC doesn't support CHIRRTL to FIRRTL dialect transforms?

Reply:
I though it's a temporary workaround for lack of fixed-point support in MFC. The current fixed-point implementation in dsptools needs to be refactored to not rely on FIRRTL fixed point.

Reply:
Oh yes that is true